hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/positive/SysDelSysUndelSpec.java

```diff
@@ -70,13 +73,15 @@ private HapiSpec distinguishesAdminPrivileges() {
                 .then(
                         systemFileDelete("misc")
                                 .payingWith(SYSTEM_UNDELETE_ADMIN)
-                                .hasPrecheck(NOT_SUPPORTED),
+                                .hasPrecheckFrom(NOT_SUPPORTED, OK)
+                                .hasKnownStatus(UNAUTHORIZED),
```

Comment:
Is UNAUTHORIZED or NOT_SUPPORTED the expected code? It looks like the existing code expected NOT_SUPPORTED from the precheck, but the new code expects UNAUTHORIZED to be determined at handle time. But NOT_SUPPORTED and UNAUTHORIZED are quite different in meeting. Should we be changing the expectation from NOT_SUPPORTED to UNAUTHORIZED?

Reply:
This test confirms that the system delete admin is not authorized to do system delete and the previous status is less accurate

Reply:
But does this mean mono-service returns one status and modular is returning a different status? I thought our goal was to maintain compatibility on status codes?

Reply:
Do we wants a special case to return NOT_SUPPORTED in this specific case (the SystemUndelete admin trying to use SystemDelete) because AUTHORIZATION_FAILED is thrown [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java#L463)

Reply:
we reverted the test back as it was with @tinker-michaelj and applied thee right logic in the workflow.